src/components/textarea/textarea.component.tsx

```diff
@@ -284,9 +294,9 @@ export const Textarea = React.forwardRef(
         <Input
           aria-invalid={!!error}
           aria-labelledby={ariaLabelledBy}
-          aria-describedby={
+          aria-describedby={`${
             validationRedesignOptIn ? undefined : ariaDescribedBy
-          }
+          }${hintDescribedBy}`}
```

Comment:
```suggestion
          } ${hintDescribedBy}`}
```
Besides I think it would be more readable you'd move the ternary conditional to a separate const like `hintDescribedBy`